src/configure/templateInputHelper/InputControl.ts

```diff
@@ -110,11 +130,41 @@ export class InputControl {
 
         if (!!properties) {
             value = properties[propertyName];
-            if( !!value && !!inputs ) { return MustacheHelper.render(value, { inputs: inputs }); }
+            if (!!value && !!inputs) { return MustacheHelper.render(value, { inputs: inputs }); }
         }
         return value;
     }
 
+    public setValidationDataSources(dataSourceToInputsMap: Map<DataSource, InputControl[]>) {
+        this.validationDataSourceToInputsMap = dataSourceToInputsMap;
```

Comment:
Do we really need this map? What purpose does it serve? Should we just a list of <DataSource> as dynamic validations for an input are already stored in `this.inputDescriptor.dynamicValidations`

Reply:
this.inputDescriptor.dynamicValidations contains only dynamicValidations source Id, this is map of datasource  and required inputControls to evaluate the dynamic validation dataSource  which is set by InputControlProvider.

Reply:
Got it.